tests/console/yast2_nfs_client.pm

```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: yast2_nfs_client module
-# G-Maintainer: Ancor Gonzalez Sosa <ancor@suse.de>
+# Summary: yast2_nfs_client module
+# Maintainer: Ancor Gonzalez Sosa <ancor@suse.de>
```

Comment:
not really. Ancor is still there to answer questions. We have much bigger problems with the Maintainer data - for example xjin and xdzhan left and nwang moved into a different team. So I wouldn't worry too much right now.